torchci/lib/types.ts

```diff
@@ -21,8 +21,8 @@ export interface JobData extends BasicJobData {
   logUrl?: string;
   durationS?: number;
   queueTimeS?: number;
-  failureLine?: string;
-  failureLineNumber?: number;
+  failureLines?: string[];
```

Comment:
Is this a correct understanding that this is only a cosmetic change atm because the current log classifier will always return only one failure line?  So this would be an array one just one string

Reply:
yep it's just cosmetic atm